bench-tps/src/cli.rs

```diff
@@ -316,30 +316,30 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
         .arg(
             Arg::with_name("rpc_client")
                 .long("use-rpc-client")
-                .conflicts_with("tpu_client")
+                .conflicts_with("thin_client")
                 .takes_value(false)
                 .help("Submit transactions with a RpcClient")
         )
         .arg(
-            Arg::with_name("tpu_client")
-                .long("use-tpu-client")
+            Arg::with_name("thin_client")
+                .long("use-thin-client")
                 .conflicts_with("rpc_client")
                 .takes_value(false)
-                .help("Submit transactions with a TpuClient")
+                .help("Submit transactions with a ThinClient")
```

Comment:
Actually, if we're going to remove this imminently, I think we should avoid exposing a flag for it in the first place.
If there is an internal need to keep it for a bit (eg `/net` scripts), let's mark this as `.hidden(hidden_unless_forced())` so it's not visible. If there's no need for it, let's just remove this Arg.

Reply:
Ya if we remove it here we'd have to remove all references to `ThinClient` in `/net` scripts in this PR as well.  So maybe as you said, let's use: `.hidden(hidden_unless_forced())` for now. And then for the PR where we will deprecate `ThinClient`, we can fully remove this flag and remove all references to `ThinClient` in `/net`. does that work?

Reply:
Yes, that works for me.